### DISCUSSION TITLE
使用finetune_hf.py进行65条数据的微调，报cuda out of memory，是需要添加什么配置的吗？

### DISCUSSION THREAD

**Discussion by zhaohai7206** [zai-org/ChatGLM3#1085] [General]
我这是两张4080卡，都是16G的，执行环境也是在conda，按照requirement里面进行的python包安装，chatglm3-6模型，从https://hf-mirror.com/THUDM/chatglm3-6b/tree/main上面下载的，但是执行python finetune_hf.py ../../AdvertiseGen  chatglm/chatglm3-6b configs/lora.yaml 进行微调的时候就会报oom
![image](https://github.com/THUDM/ChatGLM3/assets/50861760/c8c5b162-99f4-4d7e-b8af-d32e2b12f97e)
请问下，是需要配置什么的吗，代码文件没有任何修改的

**Comment by zhaohai7206** [comment]
看晚上有人使用过python finetune_hf.py  AdvertiseGen chatglm/chatglm3-6b configs/lora.yaml output，后面参数多了一个output，但是我执行的会报下面的问题，并且output目录没有任何生成任何文件
![image](https://github.com/THUDM/ChatGLM3/assets/50861760/9f151209-a062-4942-be3c-a2983a9affbb)

**Reply by zhaohai7206** [reply]
有哪位大佬帮忙解决下的吗，求助啊

**Comment by zRzRzRzRzRzRzR** [comment]
16G可能不够的话，你可以降低一下max token 16G太极限了

**Reply by zhaohai7206** [reply]
最上面输出提示里面显示需要17179869184.00G，这个是什么意思，目前应该不可能有这么大内存的吧，还是说需要设置什么降低这个内存使用需求的
